packages/wallets/src/components/Base/WalletPasswordField/WalletPasswordField.tsx

```diff
@@ -76,23 +77,19 @@ const WalletPasswordField: React.FC<WalletPasswordFieldProps> = ({
         }
     }, [isTouched]);
 
-    function getMessage() {
-        if (isTouched) {
-            if (errorMessage) {
-                return errorMessage;
-            }
-            return message;
+    useEffect(() => {
+        if (passwordError) {
+            setShowErrorMessage(true);
         }
-    }
+    }, [passwordError]);
 
     return (
         <div className='wallets-password'>
             <WalletTextField
                 autoComplete={autoComplete}
-                errorMessage={isTouched && (passwordError ? passwordErrorMessage.PasswordError : errorMessage)}
-                isInvalid={(passwordValidation && isTouched) || passwordError}
+                errorMessage={isTouched && (showErrorMessage ? passwordErrorMessage.PasswordError : errorMessage)}
```

Comment:
`passwordError` is a boolean right? Can't we just use `passwordError` here instead of a new state `shouldErrorMessage`?

Reply:
`passwordError` is passed from a response in the API. It will always be true unless there is a resubmission or etc. I want to display the error message once and when the user makes any changes to input field it will revalidate back to default. Hence why, I am using a state here.